distributed/worker.py

```diff
@@ -2103,8 +2103,8 @@ def transition_cancelled_error(
     def transition_generic_error(
         self,
         ts: TaskState,
-        exception: Exception,
-        traceback: object,
+        exception: bytes,
+        traceback: bytes,
```

Comment:
When I look at a worker's TaskState I'm seeing these as `Serialize` objects rather than `bytes`.  I would be totally fine with storing these as any of Serialize, bytes, or native Python objects.  Just calling this out here in case this wasn't your intent.

Reply:
You're right. I redid the PR in light of this.